spec/unit/deployment.spec.js

```diff
@@ -150,7 +151,7 @@ describe('Windows 10 deployment interacts with the file system as expected.', fu
 describe('Windows 8.1 deployment interacts with the file system as expected.', function() {
 
     function runMock(cmd, args, cwd) {
-        expect(cmd).toBe('c:\\Program Files (x86)\\Microsoft SDKs\\Windows Phone\\v8.1\\Tools\\AppDeploy\\AppDeployCmd.exe');
+        expect(cmd).toBe(path.join('c:', 'Program Files (x86)', 'Microsoft SDKs', 'Windows Phone', 'v8.1', 'Tools', 'AppDeploy', 'AppDeployCmd.exe'));
```

Comment:
@purplecabbage it looks like it doesn't work.
here is a Travis build for my repo with these changes:
https://travis-ci.org/alsorokin/cordova-windows/builds/79511398